### PR TITLE
Updated docs for extensibility and ability to use any languages

### PR DESCRIPTION
**Pull request by johnlafleur**
2 main changes in the documentation:
- put extensibility more forward
- inform user they will be able to build connectors in the language of their choice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Here is a [step-by-step guide](docs/getting-started-tutorial.md) showing you how
 * **Incremental updates**: Automated replications are based on incremental updates to reduce your data transfer costs.
 * **Manual full refresh**: Sometimes, you need to re-sync all your data to start again.
 * **Debugging autonomy**: Modify and debug pipelines as you see fit, without waiting.
+* **Integration extensibility:** Adapt an existing integration to your needs or build a new one with ease.
```
Comment (sherifnada) [inline README.md:44]: Two suggestions: 
1. replace `Integration extensibility` with `Built for extensibility`
1. make this the first or second bullet point in this list

File: README.md [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
-![](docs/.gitbook/assets/sources_list%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29.png)
+![](docs/.gitbook/assets/sources_list%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%282%29.png)
 
 Airbyte is on a mission to make data integration pipelines a commodity.
 
 * **Maintenance-free connectors you can use in minutes**. Just authenticate your sources and warehouse, and get connectors that adapt to schema and API changes for you.
-* Designed to **cover the long tail of integrations**, as Airbyte will be active in maintaining the pipelines’ reliability. 
-* **Building new integrations made trivial**. We make it very easy to add new integrations that you need, by offering scheduling and orchestration. 
+* Designed to **cover the long tail of integrations and needs.** Benefit from the community's battle-tested connectors and adapt them to your specific needs. 
+* **Building new integrations made trivial**. We make it very easy to add new integrations that you need, using the language of your choice, by offering scheduling and orchestration.
```
Comment (sherifnada) [inline README.md:19]: can we make this the second bullet point?

File: README.md [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
 
-![](docs/.gitbook/assets/sources_list%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29.png)
+![](docs/.gitbook/assets/sources_list%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%282%29.png)
 
 Airbyte is on a mission to make data integration pipelines a commodity.
 
 * **Maintenance-free connectors you can use in minutes**. Just authenticate your sources and warehouse, and get connectors that adapt to schema and API changes for you.
-* Designed to **cover the long tail of integrations**, as Airbyte will be active in maintaining the pipelines’ reliability. 
-* **Building new integrations made trivial**. We make it very easy to add new integrations that you need, by offering scheduling and orchestration. 
+* Designed to **cover the long tail of integrations and needs.** Benefit from the community's battle-tested connectors and adapt them to your specific needs.
```
Comment (sherifnada) [inline README.md:18]: is this bullet point needed?

**Review by sherifnada — APPROVED** [review]
LGTM with a couple of comments

File: README.md (hunk 3 above)
Comment (johnlafleur) [reply README.md:18]: I feel so tbh, given that people will always prefer to use pre-built connectors.

File: README.md (hunk 2 above)
Comment (johnlafleur) [reply README.md:19]: Done

File: README.md (hunk 1 above)
Comment (johnlafleur) [reply README.md:44]: 1. Done
2. Put in 2nd point.

**Pull request closed without merging**